vector/src/main/java/im/vector/app/features/settings/notifications/advanced/VectorSettingsAdvancedNotificationPreferenceFragment.kt

```diff
@@ -23,10 +23,15 @@ import im.vector.app.core.preference.PushRulePreference
 import im.vector.app.core.preference.VectorPreference
 import im.vector.app.core.utils.toast
 import im.vector.app.features.settings.VectorSettingsBaseFragment
+import im.vector.app.features.settings.notifications.NotificationIndex
+import im.vector.app.features.settings.notifications.StandardActions
+import im.vector.app.features.settings.notifications.getStandardAction
+import im.vector.app.features.settings.notifications.notificationIndex
 import kotlinx.coroutines.launch
 import org.matrix.android.sdk.api.session.pushrules.RuleIds
 import org.matrix.android.sdk.api.session.pushrules.rest.PushRuleAndKind
 
+// TODO This fragment seems not used anymore, we can probably delete it
```

Comment:
Is it worth doing this now while we're already making changes to notification preference screens?

Reply:
I removed it initially but it involved also removing the related resources. I preferred to revert to not include more changes in the PR and I was wondering if it could be used in f-droid (even if I did not find any usage). Wdyt @bmarty? If we can remove it, I'll handle it in a separate PR.